templates/analysis.ng.yml

```diff
@@ -77,7 +77,7 @@ steps:
       **Branch:** $(Build.SourceBranch)
       **Trigger:** $(Build.Reason)
       **Message:** $(Build.SourceVersionMessage)
-      **Report:** <${{ parameters.sonarUrlPrefix }}$(Build.SourceBranch)&id=${{ parameters.sonarKeyPrefix }}${{ parameters.microservice }}>
+      **Report:** <${{ parameters.sonarUrlPrefix }}$(Build.SourceBranchName)&id=${{ parameters.sonarKeyPrefix }}${{ parameters.microservice }}>
```

Comment:
FYI this will break if someone ever uses a slash in a branch name, because Build.SourceBranchName was broken from the start and they don't want to change it now. the issue is mentioned in the docs.